src/assets/_project/_blocks/components/forms/recaptcha.js

```diff
@@ -9,13 +9,23 @@
 import keys from '../../data/qg-google-keys';
 
 (function ($, swe) {
-  let checkEnv = window.location.hostname.search(/\bdev\b|\btest\b|\blocalhost\b|\bgithub\b|\buat\b/);
+  // let checkEnv = window.location.hostname.search(/\bdev\b|test|localhost|github|\buat\b/);
+  /*
+   * isProd function checks if keywords 'dev|test|localhost|github|uat' are not in the hostname then the environment is PROD.
+   * This may not work in all the cases.
+  */
+  let isProd = function () {
+    return window.location.hostname.search(/\bdev\b|test|localhost|github|\buat\b/) === -1;
+  };
+  isProd();
   let $feedbackForm = $('#qg-page-feedback-form');
 
   if ($feedbackForm.length > 0) {
     let setUrlEnableCaptcha = () => {
       // if environment is not PROD then use test submission handler link
-      checkEnv !== -1 ? $feedbackForm.attr('action', 'https://test.smartservice.qld.gov.au/services/submissions/email/feedback/feedback') : '';
+      if (!isProd()) {
+        $feedbackForm.attr('action', 'https://test.smartservice.qld.gov.au/services/submissions/email/feedback/feedback-v2');
```

Comment:
can we alter the domain only instead of the fully url. ie www.smartservice.qld.gov.au is altered to test.smartservice.qld.gov.au so that we miss something if the url does change.

Reply:
In the recent commit, I have made this change